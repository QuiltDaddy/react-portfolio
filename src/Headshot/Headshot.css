header{
  height: 0px;
}

.headshot {
  position: absolute;
  right: 0px;
  top: 6vw;
  width: 100%;
  height: calc(900vw / 16);
}

.tagContainer {
  display: flex;
  flex-direction: column;
  position: relative;
  top: calc(685vw / 16);
  left: 52.5%;
  width: 45%;
  z-index: 9;
}

.tagline {
  font-family: 'Montserrat Alternates', sans-serif;
  font-weight: 400;
  font-size: 4vw;
  color: #88FFFA;
}

.tagline1 {
  align-self: flex-start;
}

.tagline2 {
  position: relative;
  right: 0.75vw;
}

.tagline3 {
  align-self: flex-end;
}
