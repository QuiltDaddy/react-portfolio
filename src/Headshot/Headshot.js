import React, { Component } from 'react';
import headshot from '../pictures/headshot.jpg';
import './Headshot.css';

class Headshot extends Component {
  render() {
    return (
      <header role='banner'>
        <img
          className='headshot'
          id='headshot'
          src={headshot}
          alt='A handsome man smiles in front of a field of trees.' />
        <section className='tagContainer'>
          <p className='tagline tagline1'>
            Patrick
          </p>
          <p className='tagline tagline2'>
            purges
          </p>
          <p className='tagline tagline3'>
            puzzles.
          </p>
        </section>
      </header>
    );
  }
}

export default Headshot;
