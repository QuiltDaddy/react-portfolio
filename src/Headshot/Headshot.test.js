import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Headshot from './Headshot';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Headshot />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
const tree = renderer
  .create(<Headshot />)
  .toJSON();
expect(tree).toMatchSnapshot();  
});

