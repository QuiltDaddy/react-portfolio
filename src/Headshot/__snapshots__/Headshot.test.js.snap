// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<header
  role="banner"
>
  <img
    alt="A handsome man smiles in front of a field of trees."
    className="headshot"
    id="headshot"
    src="headshot.jpg"
  />
  <section
    className="tagContainer"
  >
    <p
      className="tagline tagline1"
    >
      Patrick
    </p>
    <p
      className="tagline tagline2"
    >
      purges
    </p>
    <p
      className="tagline tagline3"
    >
      puzzles.
    </p>
  </section>
</header>
`;
